keyboards/atlantis/ak81_ve/readme.md

```diff
@@ -0,0 +1,22 @@
+# ATLANTIS AK81_VE
+
+![ATLANTIS AK81_VE](https://i.imgur.com/nfuocgS.jpeg)
+
+A 75% hotswap keyboard with per key RGB backlighting, RGB underglow and a rotary encoder.
+
+* Keyboard Maintainer: [fOmey](https://github.com/fOmey)
+* Hardware Supported: AK75X@AK81_VE
+* Hardware Availability: [AliExpress.com](https://aliexpress.com), [Shopee](https://shopee.com.my)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make atlantis/ak81_ve:default
+
+## Bootloader
+
+Enter the bootloader in 2 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix
+* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Comment:
Does the PCB have some kind of reset button or pads to be shorted with tweezers?

Reply:
Turns out it does, discovered a unpopulated reset button pad while checking LED count.